frontier_explorer:
  ros__parameters:
    map_topic: /map
    use_sim_time: false
    base_frame: base_link
    linear_speed: 0.8
    angular_speed: 0.8
    goal_tolerance: 0.25
    min_frontier_neighbors: 1
    timer_period: 0.5
    safety_stop_distance: 0.40
    safety_angle_deg: 30.0
